README.md

```diff
@@ -98,5 +98,5 @@ In order to build and use `SNAPSHOT.jar` package version refer to the following
 
 ## Backward Compatibility
 
-Version 0.7.0 is not backward compatible with any previous versions.
-Due to a change in the consensus protocol, you cannot update any existing Sidechain of version <=0.6.0 into a >=0.7.0
\ No newline at end of file
+Version 0.8.0 is not backward compatible with any previous versions.
```

Comment:
We should use something like 'Any version >= 0.7.0 is not backward compatible', and the same below, we should keep the old comment.

Reply:
I have used `/ci/devtools/set_version.sh` script and it only changes VERSION number in various places. The exact wording will still need to be changed manually by someone who knows what exactly it should be